accounts/lists.py

```diff
@@ -0,0 +1,27 @@
+# User types list passed into dropdown of same name for user selection used
+# in models.py & forms.py
+user_types = [
```

Comment:
I would make `user_types` and `lms_modules` constants. Also, in the documentations in Django it would be suffixed with `_CHOICES` which would be `USER_TYPE_CHOICES` and `LMS_MODULES_CHOICES`.